app/services/editions/SQSFacade.scala

```diff
@@ -0,0 +1,41 @@
+package services.editions
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder
+import com.amazonaws.services.sqs.model.{Message, ReceiveMessageRequest}
+import conf.ApplicationConfiguration
+import logging.Logging
+
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+object SQSFacade {
```

Comment:
Would `SQSClient` be more intention revealing?

Reply:
i changed name of that class to SQSIssuePublishEventsQueueFacade because it will be specific for messages from issue events queue only

Reply:
that is why it is facade because it will hide all the complexity related to reading from that specific queue not generic one like max messages or wait-time